src/ripple/app/ledger/LedgerTiming.cpp

```diff
@@ -128,7 +128,7 @@ bool ContinuousLedgerTiming::haveConsensus (
     }
 
     // If 80% of current proposers (plus us) agree on a set, we have consensus
-    if (((currentAgree * 100 + 100) / (currentProposers + 1)) > 80)
+    if (((currentAgree * 100 + 100) / (currentProposers + 1)) >= CONSENSUS_PCT)
```

Comment:
I want to make sure I understand the intent.  Because we have exactly 5 validators, the old code (with `>`) required that all five validators agree in order to have "normal consensus".  By changing from `>` to `>=` we're saying that "normal consensus" succeeds if 4 out of 5 validators agree.  If that's the intent I believe this change accomplishes the intent.

I personally prefer that we do this computation in integer math.  Integer math gives me greater confidence that two validators will come to the same conclusion given the same data.  Floating point gets blurry on edge cases (like this one that Mark is addressing.